src/selectors/disabled.test.js

```diff
@@ -15,11 +17,67 @@ const getEdgeTargets = state => state.edge.targets;
 const getNodeTags = state => state.node.tags;
 
 describe('Selectors', () => {
-  describe('getNodeDisabledTag', () => {
-    it('returns an object', () => {
-      expect(getNodeDisabledTag(mockState.animals)).toEqual(expect.any(Object));
+  describe('getNodeDisabledPipeline', () => {
+    it("returns an object whose keys match the current pipeline's nodes", () => {
+      expect(Object.keys(getNodeDisabledPipeline(mockState.animals))).toEqual(
+        getNodeIDs(mockState.animals)
+      );
+    });
+
+    it('returns an object whose values are all Booleans', () => {
+      expect(
+        Object.values(getNodeDisabledPipeline(mockState.animals)).every(
+          value => typeof value === 'boolean'
+        )
+      ).toBe(true);
+    });
+
+    it('does not disable any nodes if there is no active pipeline', () => {
+      const activePipeline = undefined;
+      const newMockState = reducer(
+        mockState.animals,
+        updateActivePipeline(activePipeline)
+      );
+      const nodeDisabledPipeline = getNodeDisabledPipeline(newMockState);
+      expect(
+        mockState.animals.node.ids.every(
+          nodeID => !nodeDisabledPipeline[nodeID]
+        )
+      ).toBe(true);
+    });
+
+    it('does not disable any nodes that are in the active pipeline', () => {
```

Comment:
Might be clearer to flip the wording to positives e.g. `active pipeline nodes remain enabled when...`?

Reply:
Tempted to leave this, just because I was trying to keep the tense consistent so that it works with the 'it' statement. Could go either way though 🤷